cmd/subctl/cloud.go

```diff
@@ -34,6 +34,7 @@ const (
 
 var (
 	cloudPorts cloud.Ports
+	useLB      bool
```

Comment:
Let's encapsulate this in a struct for clarity and to isolate it from other global vars in the package.

```suggestion
        cloudOptions struct {
		useLoadBalancer      bool
	}
```